lib/slack/web/api/errors/slack_error.rb

```diff
@@ -10,6 +10,18 @@ def initialize(message, response = nil)
             super message
             @response = response
           end
+
+          def error
```

Comment:
This is sort of a horrible name. I used it because it's the name of the field in the response body, but I'm open to changing it. One alternative is `error_code` (and `error_codes` below).